src/client/components/common/UseCaseWrapper/UseCaseWrapper.tsx

```diff
@@ -0,0 +1,134 @@
+import { ElementRef, FunctionComponent, useRef } from 'react';
+import Container from '../Container';
+import styles from './UseCaseWrapper.module.scss';
+import Button from '../Button';
+import Lightbulb from './lightbulb.svg';
+import Image from 'next/image';
+import { Paper } from '@mui/material';
+import { UseCase } from '../content';
+import ExternalLinkIcon from './externalLinkArrow.svg';
+
+type UseCaseWrapperProps = {
+  title: string;
+  description?: React.ReactNode;
+  articleURL?: string;
+  listItems?: readonly React.ReactNode[];
+  children: React.ReactNode;
+  hideSrcListItem?: boolean;
+  hideDivider?: boolean;
+  mentionResetButton?: boolean;
+  returnUrl?: string;
+  contentSx?: React.CSSProperties;
+  moreResources?: UseCase['moreResources'];
+};
```

Comment:
Question - wouldn't it be more efficient just pass `UseCase` to the props?

Reply:
Yeah, I was thinking about it, but it's also kind of nice to have the flexibility to assemble it Lego-style. I can make Playground using UseCaseWrapper, without modeling it as a use case (which it isn't). But if you prefer I can do that, or simplify it later.